examples/equilibrium_temperature_co2.jl

```diff
@@ -0,0 +1,30 @@
+using Klimakoffer
+using Printf
+
+NT = 48 # Number of time-steps per year
+
+mesh = Mesh()
+
+model = Model(mesh, NT)
+
+discretization = Discretization(mesh, model, NT)
+
+year_start = 1950
+year_end = 2020
+year_delta = year_end-year_start+1
+
+co2_concentration_yearly = zeros(Float64,year_delta)
+mean_temperature_yearly = zeros(Float64, year_delta)
```

Comment:
```suggestion
co2_concentration_yearly = zeros(year_delta)
mean_temperature_yearly = zeros(year_delta)
```
No explicit types where not needed.

Reply:
We decided to keep the Float64, although not needed throughout the code